cis194/week4/daniel-deng/src/Exercise04.hs

```diff
@@ -0,0 +1,16 @@
+module CIS194.Homework04.Exercise04 (sieveSundaram) where
+
+cartProd :: [a] -> [(a, a)]
+cartProd xs = [(a, b) | a <- xs, b <- xs]
+
+validNumber :: Integer -> Bool
+validNumber x = not
```

Comment:
@shineli1984 any idea how we can implement this in a point-free style here? Does point-free style make sense here from a pragmatic perspective?

Reply:
I think this is good enough to convey the intention.